### PR TITLE
Implement hidden headings (Accessibility)

### PR DESCRIPTION
**Pull request by maethu**

### CODE REVIEW

**Comment by maethu** [comment]
Looks now like:

screen shot 2015-09-08 at 16 51 04

File: plonetheme/onegovbear/theme/index.html [hunk 1]
```diff
@@ -217,6 +218,7 @@ <h1 class="documentFirstHeading">This is a heading 1</h1>
               <!-- closes column-content -->
             </div>
 
+            <h1 class="hiddenStructure">Weitere Informationen</h1>
```
Comment (jone) [inline plonetheme/onegovbear/theme/index.html]: Can we somehow make this translatable?

Comment (maethu) [inline plonetheme/onegovbear/theme/index.html]: Afaik no. We may implement it into the accesibility viewlet? and move it with diazo to the right position?? imho not mandatory...

Comment (jone) [inline plonetheme/onegovbear/theme/index.html]: i'd prefer that

Comment (maethu) [inline plonetheme/onegovbear/theme/index.html]: awwww...

**Pull request merged**